Local Devnet macOS Tahoe Docker Prereq Update (#49)
datahaven-xyz/datahaven-docs: store-and-retrieve-data/spin-up-local-devnet.md

- macOS or Linux operating system
- [Docker](https://www.docker.com/){target=\_blank} installed and running

    ??? interface "Instructions if running macOS Tahoe (v26 or higher)"

>         If running macOS Tahoe (v26 or higher), instead of using Docker, you should install [OrbStack](https://orbstack.dev/download){target=\_blank}. Make sure Docker Desktop is not running while you are using OrbStack in order to allow OrbStack to work properly. You can run all your docker commands the same way while using OrbStack. 

Reviewer: ```suggestion
        If running macOS Tahoe (v26 or higher), instead of using Docker, you should install [OrbStack](https://orbstack.dev/download){target=\_blank}. Make sure Docker Desktop is not running while using OrbStack to ensure it works properly. You can run all your Docker commands the same way while using OrbStack. 
```

>         After installing OrbStack to confirm you are using OrbStack under the hood instead of Docker Desktop you can run the following command in terminal:

Reviewer: This is valid
